app/src/main/java/org/sopt/and/core/designsystem/component/textfield/PasswordTextField.kt

```diff
@@ -0,0 +1,69 @@
+package org.sopt.and.core.designsystem.component.textfield
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.OutlinedTextFieldDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+import androidx.compose.ui.unit.dp
+import org.sopt.and.R
+
+@Composable
+fun PasswordTextField(
+    password: MutableState<String>,
+    placeholder: String,
+    isVisible: MutableState<Boolean>,
+    onValueChange: (String) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    OutlinedTextField(
+        modifier = modifier
+            .fillMaxWidth(),
+        value = password.value,
+        onValueChange = { newValue ->
+            password.value = newValue
+            onValueChange(newValue)
+        },
```

Comment:
```suggestion
        onValueChange = { newValue ->
            password.value = newValue
        },
```

이렇게 줄일 수 있도록 구조 변경이 있어야할 것 같네요.